config/common/common.go

```diff
@@ -33,9 +35,14 @@ var (
 	snakeRe = regexp.MustCompile("([A-Z])")
 )
 
+type BaseOptions = base.TranslateOptions
+type FCOSOptions = fcos.TranslateOptions
+
 type TranslateOptions struct {
 	Pretty bool
 	Strict bool
+	BaseOptions
+	FCOSOptions
```

Comment:
I think this uninhabited and currently unused, embedded here only for future usage. Thus I'm wondering, how do you plan to wire distro-specific options to the CLI? Wouldn't it be enough to have a single set of options, mirroring what the CLI exposes (and only using the relevant subset in each distro)?

I know this fully-customizable approach looks enticing, but I'm wary of inducing a mess at the CLI-flags level.

Reply:
I was expecting all options to be wired into the CLI equally, without distinguishing between distro-specific and generic ones.  You're right that that might have been confusing.  I've dropped the distro-specific options struct.